docker-compose-local.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
 
   localstack:
-    image: localstack/localstack:2.3
+    image: localstack/localstack:3
```

Comment:
No breaking changes?

Reply:
read-me said nothing that sounded significant 🤷